main.bat

```diff
@@ -0,0 +1,30 @@
+@echo off
+echo Killing an running versions of this script...
+taskkill /F /FI "windowtitle eq Plex - Trakt Sync" /T
+
+SET mypath=%~dp0
+cd %mypath%\
+
+cls
+
+:: Check for Python Installation
+py --version 2>NUL
+if errorlevel 1 goto errorNoPython
+
+cls
+
+title Plex - Trakt Sync
+
+
+py main.py
```

Comment:
```suggestion
py -m plex_trakt_sync
```

`main.py` is deprecated, should be using module mode:
- https://github.com/Taxel/PlexTraktSync#setup

you should also rename your script accordingly.

Reply:
the wrapper must also accept extra arguments, for example `py -m plex_trakt_sync watch`